frontend/src/metabase/components/ThinProgressBar.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from "react";
+
+import "./ThinProgressBar.css";
```

Comment:
I would just use `cxs` here. The amount of CSS and Component code is small enough that separating this out will make it harder to parse what's going on.